run_tests.sh

```diff
@@ -9,6 +9,9 @@ export TF_CPP_MIN_LOG_LEVEL="3"
 pytest -q tests/attacks/evasion/ --mlFramework="tensorflow" --durations=0
 if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed attacks/evasion tests"; fi
 
+pytest -q -s tests/attacks/evasion/test_shadow_attack.py --mlFramework="pytorch" --durations=0
```

Comment:
these two lines should be moved below line 32 as `pytest -q -s tests/attacks/evasion/test_shadow_attack.py --mlFramework=$mlFRamework --durations=0` in order to make sure this issue of not being run with the correct framework does not occur again

Reply:
As discussed offline, we'll keep lines 12 and 13 for now to avoid running the test twice.